lambdas/data-migration2/src/executions.ts

```diff
@@ -80,34 +121,26 @@ export const migrateExecutions = async (
     skipped: 0,
   };
 
-  let record = await searchQueue.peek();
-  /* eslint-disable no-await-in-loop */
-  while (record) {
-    migrationResult.total_dynamo_db_records += 1;
-
-    if (migrationResult.total_dynamo_db_records % loggingInterval === 0) {
-      logger.info(`Batch of ${loggingInterval} execution records processed, ${migrationResult.total_dynamo_db_records} total`);
-    }
-
-    try {
-      await migrateExecutionRecord(<ExecutionRecord>record, knex);
-      migrationResult.migrated += 1;
-    } catch (error) {
-      if (error instanceof RecordAlreadyMigrated) {
-        migrationResult.skipped += 1;
-      } else {
-        migrationResult.failed += 1;
-        logger.error(
-          `Could not create execution record in RDS for Dynamo execution arn ${record.arn}:`,
-          error
-        );
-      }
-    }
+  const totalSegments = executionMigrationParams.parallelScanSegments ?? 5;
```

Comment:
Totally arbitrary. I set these defaults before doing any extensive testing so I wasn't sure how much we would be able to get away with. However, the appropriate value to use here is really going to demand on how much data you have and how far you're willing to push your database. Still, we could probably set it to something more aggressive like `20` since I've just done tests with this value set to `50` and it didn't tax the database that much (~60% CPU usage, 2 ACUs)

Reply:
Cool, thanks for the clarification.   Yes, that seems like a decent enough change if it's staying at 2ACU, but I'm fine with being conservative regarding this as well.

Reply:
21dc1a5a858f3832fd05dee8b67fa74b156e3f36